sourcecode/apis/contentauthor/resources/assets/react/components/QuestionContentType/components/QuestionCard/components/AddCard/index.js

```diff
@@ -17,18 +19,23 @@ const AddCard = props => {
             fontWeight: '400 !important',
         }
     };
+    const useStyles = makeStyles((theme) => ({
```

Comment:
Avoid using several different ways of styling. Use either the current or the new, not both. So either move `addNewQuestionContainer` to `useStyles` or move `largeButton` to `styles`.